src/components/ha-form/ha-form.ts

```diff
@@ -45,6 +47,11 @@ export interface HaFormSelectSchema extends HaFormBaseSchema {
   options?: string[];
 }
 
+export interface HaFormMultiSelectSchema extends HaFormBaseSchema {
+  type: "multi_select";
+  options?: string[];
```

Comment:
Backend support has been added https://github.com/home-assistant/home-assistant/pull/31798

This has become dictionary of key=>label: `{ [key: string]: string }